kolibri/core/content/apps.py

```diff
@@ -15,4 +18,10 @@ def ready(self):
         from kolibri.core.content.utils.sqlalchemybridge import prepare_bases
         from .signals import cascade_delete_node  # noqa: F401
 
+        # Do this to prevent import of broken Windows filetype registry that makes guesstype not work.
+        # https://www.thecodingforums.com/threads/mimetypes-guess_type-broken-in-windows-on-py2-7-and-python-3-x.952693/
+        mimetypes.init(
```

Comment:
Hrm, realizing that this may be the wrong place for this, as we are mostly depending on it outside of the Django lifecycle, should revisit.